app/src/main/java/com/kickstarter/ui/activities/SignupActivity.kt

```diff
@@ -1,130 +1,87 @@
 package com.kickstarter.ui.activities
 
 import android.os.Bundle
+import androidx.activity.compose.setContent
 import androidx.activity.viewModels
 import androidx.appcompat.app.AppCompatActivity
-import androidx.core.widget.doOnTextChanged
-import com.kickstarter.R
-import com.kickstarter.databinding.SignupLayoutBinding
-import com.kickstarter.libs.utils.SwitchCompatUtils
-import com.kickstarter.libs.utils.ViewUtils
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material.rememberScaffoldState
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.rxjava2.subscribeAsState
+import com.kickstarter.libs.featureflag.FlagKey
 import com.kickstarter.libs.utils.extensions.addToDisposable
 import com.kickstarter.libs.utils.extensions.getEnvironment
-import com.kickstarter.ui.extensions.hideKeyboard
-import com.kickstarter.ui.views.LoginPopupMenu
+import com.kickstarter.ui.activities.compose.login.SignupScreen
+import com.kickstarter.ui.compose.designsystem.KickstarterApp
+import com.kickstarter.ui.extensions.startDisclaimerActivity
 import com.kickstarter.viewmodels.SignupViewModel
 import io.reactivex.android.schedulers.AndroidSchedulers
 import io.reactivex.disposables.CompositeDisposable
 
 class SignupActivity : AppCompatActivity() {
 
-    private lateinit var binding: SignupLayoutBinding
     private lateinit var viewModelFactory: SignupViewModel.Factory
     private val viewModel: SignupViewModel.SignupViewModel by viewModels { viewModelFactory }
     private val disposables = CompositeDisposable()
+    var darkModeEnabled = false
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
         this.getEnvironment()?.let { env ->
             viewModelFactory = SignupViewModel.Factory(env)
+            darkModeEnabled = env.featureFlagClient()?.getBoolean(FlagKey.ANDROID_DARK_MODE_ENABLED) ?: false
```

Comment:
Small change needed here:
If this is true it will always show dark mode rather than using the system setting, do something like this:
`useDarkTheme = if (darkModeEnabled) isSystemInDarkTheme() else false`

Reply:
If you take a look at line 51, we are checking for both the dark mode feature flag as well as if the system is in dark mode

Reply:
oh woops